README.rst

```diff
@@ -1320,21 +1320,18 @@ Example usage:
     cd tests/
     ./manage.py run_checks
 
-Monitoring scripts
+Monitoring Package
```

Comment:
I believe this information should come much earlier in the README, we should probably move it up (eg: what about moving it before https://github.com/openwisp/openwisp-monitoring/tree/issues/298-receive-data-when-device-was-offline#device-health-status?

And what if we move https://github.com/openwisp/openwisp-monitoring#setup-integrate-in-an-existing-django-project) just after the development install instructions?